apps/yapms/src/lib/utils/luma.ts

```diff
@@ -6,6 +6,25 @@ function hexToRGB(hex: string) {
 	return { r, g, b };
 }
 
+function blendHexForLuma(hexes: string[]) {
```

Comment:
This could probably be renamed to just "blendHexColor" or something.

Also, I think it's okay to return a Hex color with the # at the beginning, as all the other functions here account for it.